phoenix-core/src/it/java/org/apache/phoenix/end2end/TrimbFunctionIT.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.end2end;
+
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+import static org.junit.Assert.*;
+
+public class TrimbFunctionIT extends ParallelStatsDisabledIT {
+
+    private static final String KEY = "key";
+
+    private String initTable(Connection conn, byte[] b, int binaryLen, byte[] vb) throws Exception {
+        String trimbTestTableName = generateUniqueName();
+        String ddl = "CREATE TABLE " + trimbTestTableName
+                        + " (pk VARCHAR NOT NULL PRIMARY KEY , b BINARY(" + binaryLen
+                        + "), vb VARBINARY)";
+        conn.createStatement().execute(ddl);
+        String dml = "UPSERT INTO " + trimbTestTableName + " VALUES(?,?,?)";
+        PreparedStatement stmt = conn.prepareStatement(dml);
+        stmt.setString(1, KEY);
+        stmt.setBytes(2, b);
+        stmt.setBytes(3, vb);
+        stmt.execute();
+        conn.commit();
+        return trimbTestTableName;
+    }
+
+    @Test public void testTrimLeft() throws Exception {
+        Connection conn = DriverManager.getConnection(getUrl());
+        String val = "ABCD";
+        byte[] b = val.getBytes();
+        byte[] vb = val.getBytes();
+        String tableName = initTable(conn, b, 4, vb);
+        ResultSet rs = conn.createStatement().executeQuery("SELECT TRIMB(b,cast('AB' as VARBINARY)),TRIMB(vb,cast('A' as VARBINARY)) FROM "
```

Comment:
line more than 100 chars, plz follow the code conventions
FYI: https://phoenix.apache.org/contributing.html